utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -105,12 +105,9 @@ class HttpFileStorage(
 
         val response = httpClient.execute(request)
         if (response.isSuccessful) {
-            response.body?.let { body ->
+            response.body.let { body ->
```

Comment:
Drop this, see above.

Reply:
Fixed with refactor.

Reply:
Only in the first commit (won't compile otherwise), gone in the HEAD.